test/t/unit/test_unit_known_hosts.py

```diff
@@ -1,7 +1,6 @@
 from itertools import chain
 
 import pytest
-
```

Comment:
Is this change intentional?

Reply:
sorry, this one I missed while git diff'ing - fixed